### DISCUSSION TITLE
Issue Setting Initial Water Level in DualSPHysics Reservoir Simulation

### DISCUSSION THREAD

**Discussion by MaryamBC** [DualSPHysics/DualSPHysics#276] [DUALSPHYSICS_V5.2]
Dear DualSPHysics users,

I have a question regarding setting the initial water level for my Dualsphysics simulation.
I am modelling a reservoir with a desired starting level at El. 573 m.
The simulation uses a 5-metre particle size. Below is the portion of the code for setting the fluid:
This results in a starting reservoir level of El. 572.5 m when exporting the Gauge data generated by the model.
I have tried changing the size z from 573 to 576 with no effect on the starting level in the results.

                  <setmkfluid mk="0" />
                    <fillbox x="1638.7" y="-900" z="553">
                        <modefill>void</modefill>
                        <point x="100" y="-2200" z="0" />
                        <size x="2800" y="4050" z="573" />
                    </fillbox>
                    <_pointsmkout file="fluid" />
                    <shapeout file="" />
Your assistance would be greatly appreciated!

**Comment by Claxomere** [comment]
Hi,
Your water level difference is less than dp (576-573=3m). If you need such accuracy on the water level, then your dp is not good enough.
FYI, DPSH generate particles on a cartesian grid spaced by dp (particle ditance).

Bye
